harvard-lts/CURIOSity
# Uri bump

**Bump URI and add DS_Store to gitignore**
* * *

# What does this Pull Request do?
Bumps uri gem and adds DS_Store to gitignore

# How should this be tested?

Spin up this branch and make sure you can still do basic functions (view, login, harvest, make an exhibit).

DS_Store should no longer show in a git status

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests?
- integration tests?

# Interested parties
Tag (@ mention) interested parties
